src/plugins/gstreamer/camerabin/camerabinimageprocessing.cpp

```diff
@@ -309,14 +313,20 @@ bool CameraBinImageProcessing::isParameterValueSupported(QCameraImageProcessingC
             g_object_get(G_OBJECT(m_session->cameraSource()), "supported-color-tones", &supported_effects, NULL);
 
             if (supported_effects) {
```

Comment:
I think this `supported_effects` thing has to be unref'd too

Reply:
Yes, this is indeed a memory leak, I found similar leaks also in some other files. I will fix all of those.